### PR TITLE
Remove extra field.

### LINKED ISSUE #3467: Storaged crash when scanEdge called
**Please check the FAQ documentation before raising an issue**

Storaged crash when scan edge with the following parameter:

```golang
	req := &storage.ScanEdgeRequest{
		SpaceID: globalSpaceID,
		Parts: map[int32]*storage.ScanCursor{
			globalPartitionID: {
				HasNext:    true,
				NextCursor: nil,
			},
		},
...
```

the crash stack:

```txt
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
--Type <RET> for more, q to quit, c to continue without paging--c
Core was generated by `/root/src/nebula/build/bin/nebula-storaged --flagfile /root/nebula-chaos-cluste'.
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x0000000004f15958 in std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) ()
[Current thread is 1 (Thread 0x7f873f1ff700 (LWP 154))]
(gdb) bt
#0  0x0000000004f15958 in std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) ()
#1  0x0000000002d13411 in nebula::storage::ScanEdgeProcessor::runInSingleThread (this=0x7f8770e7d600, req=...) at /root/src/nebula/src/storage/query/ScanEdgeProcessor.cpp:139
#2  0x0000000002d125cd in nebula::storage::ScanEdgeProcessor::doProcess (this=0x7f8770e7d600, req=...) at /root/src/nebula/src/storage/query/ScanEdgeProcessor.cpp:53
#3  0x0000000002d11e81 in nebula::storage::ScanEdgeProcessor::<lambda()>::operator()(void) const (__closure=0x7f8770e211c0) at /root/src/nebula/src/storage/query/ScanEdgeProcessor.cpp:19
#4  0x0000000002d19073 in folly::detail::function::FunctionTraits<void()>::callBig<nebula::storage::ScanEdgeProcessor::process(const nebula::storage::cpp2::ScanEdgeRequest&)::<lambda()> >(folly::detail::function::Data &) (p=...)
    at /root/src/nebula/build/third-party/install/include/folly/Function.h:385
#5  0x000000000457b8a7 in virtual thunk to apache::thrift::concurrency::FunctionRunner::run() ()
#6  0x00000000046b0ca3 in apache::thrift::concurrency::ThreadManager::Impl::Worker::run() ()
#7  0x00000000046b48fd in apache::thrift::concurrency::PthreadThread::threadMain(void*) ()
#8  0x00007f877f03f609 in start_thread (arg=<optimized out>) at pthread_create.c:477
#9  0x00007f877ef66293 in clone () at ../sysdeps/unix/sysv/linux/x86_64/clone.S:95
(gdb) up
#1  0x0000000002d13411 in nebula::storage::ScanEdgeProcessor::runInSingleThread (this=0x7f8770e7d600, req=...) at /root/src/nebula/src/storage/query/ScanEdgeProcessor.cpp:139
139	    auto ret = plan.go(partId, cursor.get_has_next() ? *cursor.get_next_cursor() : "");
(gdb) down
#0  0x0000000004f15958 in std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) ()
(gdb) up
#1  0x0000000002d13411 in nebula::storage::ScanEdgeProcessor::runInSingleThread (this=0x7f8770e7d600, req=...) at /root/src/nebula/src/storage/query/ScanEdgeProcessor.cpp:139
139	    auto ret = plan.go(partId, cursor.get_has_next() ? *cursor.get_next_cursor() : "");
(gdb) p partId
$1 = 1
(gdb) p cursor.get_has_next()
You can't do that without a process to debug.
(gdb) p cursor
$2 = {static __fbthrift_cpp2_gen_json = false, static __fbthrift_cpp2_gen_nimble = false, static __fbthrift_cpp2_gen_has_thrift_uri = false, static __fbthrift_cpp2_is_union = false, has_next = true, next_cursor = {static npos = 18446744073709551615,
    _M_dataplus = {<std::allocator<char>> = {<__gnu_cxx::new_allocator<char>> = {<No data fields>}, <No data fields>}, _M_p = 0x7f873f1fc828 ""}, _M_string_length = 0, {_M_local_buf = '\000' <repeats 15 times>, _M_allocated_capacity = 0}}, __isset = {has_next = true,
    next_cursor = false}}
(gdb)
```

**Your Environments (__required__)**

* OS: `uname -a`
* Compiler: `g++ --version` or `clang++ --version`
* CPU: `lscpu`
* Commit id (e.g. `a3ffc7d8`)

**How To Reproduce(__required__)**

1. Step 1
2. Step 2
3. Step 3

**Expected behavior**

**Additional context**

### PR DESCRIPTION
**Pull request by Shylock-Hg**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### Which issue(s)/PR(s) this PR relates to?
Close https://github.com/vesoft-inc/nebula/issues/3421
Close #3467 
  
#### Checklist：
- [ ] Documentation affected （Please add the label if documentation needs to be modified.)
- [ ] Incompatible （If it is incompatible, please describe it and add corresponding label.）
- [ ] Need to cherry-pick （If need to cherry-pick to some branches, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes：
Please confirm whether to reflect in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Comment by Shylock-Hg** [comment]
Close https://github.com/vesoft-inc/nebula/issues/3421

**Comment by Aiee** [comment]
Does this PR require modification from storage clients?

**Comment by Shylock-Hg** [comment]
> Does this PR require modification from storage clients?

Yes.

File: src/interface/storage.thrift [hunk 1]
```diff
@@ -561,9 +561,8 @@ struct LookupAndTraverseRequest {
  */
 
 struct ScanCursor {
-    3: bool                                 has_next,
     // next start key of scan, only valid when has_next is true
```
Comment (Nicole00) [inline src/interface/storage.thrift:564]: how client knows the scan has get to end？

Comment (Shylock-Hg) [reply src/interface/storage.thrift:564]: When 'next_cursor' is null.

**Comment by Shylock-Hg** [comment]
Close #3467

**Review by yixinglu — APPROVED** [review]
LGTM

**Review by jievince — APPROVED** [review]

**Pull request merged**